submissions/onyemeMichael/to-do-app/src/components/TodoList.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect} from 'react';
+import TodoForm from './TodoForm';
+import Todo from './Todo';
+
+function TodoList() {
+  const [todosLeft,setTodosRemaining] = useState(0);
+  const [todos, setTodos] = useState([]);
+  useEffect(() => {
+    setTodosRemaining(todos.filter(todo => !todo.isCompleted).length);
+    
+  },
+[ todos])
+   
+
+  const addTodo = todo => {
+    if (!todo.text || /^\s*$/.test(todo.text)) {
```

Comment:
I think numbers can be included in a todo item, not just strings.

Reply:
It also allows numbers too.

Reply:
What is the test for then?